<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1250</width>
    <height>680</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTableWidget" name="tableWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>140</y>
      <width>1220</width>
      <height>510</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 700 13pt &quot;Courier&quot;;</string>
    </property>
    <attribute name="horizontalHeaderVisible">
     <bool>true</bool>
    </attribute>
    <attribute name="horizontalHeaderCascadingSectionResizes">
     <bool>false</bool>
    </attribute>
    <attribute name="verticalHeaderCascadingSectionResizes">
     <bool>true</bool>
    </attribute>
    <column>
     <property name="text">
      <string notr="true">Codigo</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Descricao                     </string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Saldo</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Preco</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Preco Aprazo</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Unidade</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Codigo Barra</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Referencia</string>
     </property>
    </column>
   </widget>
   <widget class="QLabel" name="tit_produto">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>10</y>
      <width>930</width>
      <height>60</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 700 16pt &quot;Segoe UI&quot;;</string>
    </property>
    <property name="text">
     <string>Produtos</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_pesquisa">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>80</y>
      <width>130</width>
      <height>30</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>10</pointsize>
      <italic>false</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>Pesquisa Geral:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="pesquisa">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>80</y>
      <width>301</width>
      <height>30</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);
border-color: rgb(132, 168, 163);
border-style: Outset;
border-width: 2px;
border-radius: 0px;


</string>
    </property>
    <property name="maxLength">
     <number>40</number>
    </property>
   </widget>
   <widget class="QPushButton" name="bt_inclusao">
    <property name="geometry">
     <rect>
      <x>760</x>
      <y>80</y>
      <width>100</width>
      <height>30</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>Inclusao</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="rb_alteracao">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>80</y>
      <width>100</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>Alteracao</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="rb_consulta">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>80</y>
      <width>100</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>Consulta</string>
    </property>
   </widget>
   <widget class="QPushButton" name="bt_sair">
    <property name="geometry">
     <rect>
      <x>880</x>
      <y>80</y>
      <width>100</width>
      <height>30</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>Sair</string>
    </property>
   </widget>
   <widget class="QLabel" name="empresa">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 81 10pt &quot;Rockwell Extra Bold&quot;;

background-color: rgb(223, 213, 255);

border-color: rgb(162, 162, 162);

border-style: Outset;
border-width: 2px;
border-radius: 12px;</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>CADASTRO.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="imagem_produto">
    <property name="geometry">
     <rect>
      <x>1060</x>
      <y>10</y>
      <width>170</width>
      <height>120</height>
     </rect>
    </property>
    <property name="text">
     <string>        Foto do Produto</string>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <tabstops>
  <tabstop>pesquisa</tabstop>
  <tabstop>rb_alteracao</tabstop>
  <tabstop>rb_consulta</tabstop>
  <tabstop>bt_inclusao</tabstop>
  <tabstop>bt_sair</tabstop>
  <tabstop>tableWidget</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
